process/best_practices_badge.md

```diff
@@ -0,0 +1,57 @@
+- [Secure Software Development: Implementation (LFD105x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-implementation-lfd105/) 
+- [Secure Software Development: Verification and More Specialized Topics (LFD106x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-verification-and-more-specialized-topics-lfd106/) 
+
+## Is there a list or document which describes the static analysis tools that ASWF supports or recommends?
+
+SonarCloud is the tool that multiple projects have implemented, and there is already an organization-level access token that projects can use with it. You can see a specific example in the OpenEXR analysis_workflow.yml, which shows how to build, run the test suite, post-process, and upload to SonarCloud using the SONAR_TOKEN from the organization level. You can see the results for the various ASWF projects with SonarCloud here:
+
+[https://sonarcloud.io/organizations/academysoftwarefoundation/projects](https://sonarcloud.io/organizations/academysoftwarefoundation/projects)
+
+I'm also seeing some work towards using clang-tidy in Imath: [https://github.com/AcademySoftwareFoundation/Imath/blob/main/config/ImathSetup.cmake](https://github.com/AcademySoftwareFoundation/Imath/blob/main/config/ImathSetup.cmake)
+
+and in OSL:
+
+[https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/blob/3bd377861087f99c6c69d73a95176d88072ff5da/src/cmake/compiler.cmake#L406]https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/blob/3bd377861087f99c6c69d73a95176d88072ff5da/src/cmake/compiler.cmake#L406)
+
+## "The project MUST have a documented process for responding to vulnerability reports." - does GitHub have anything to help with this?
+
+
+
+## "The project MUST list external dependencies in a computer-processable way." - is there an automated way to build this?
+
+LFX Security automatically will pull the list of dependencies for a project provided they use the standard mechanisms leveraged by Snyk. Note there is still outstanding work to be done for [C/C++ projects to be supported](https://docs.snyk.io/scan-application-code/snyk-code/snyk-code-language-and-framework-support).
+
+## "The project MUST provide an assurance case that justifies why its security requirements are met. The assurance case MUST include: a description of the threat model, clear identification of trust boundaries, an argument that secure design principles have been applied, and an argument that common implementation security weaknesses have been countered."
+
+
+
+## The project MUST have a [reproducible build](https://reproducible-builds.org/). If no building occurs (e.g., scripting languages where the source code is used directly instead of being compiled), select "not applicable"
```

Comment:
>If no building occurs (e.g., scripting languages where the source code is used directly instead of being compiled), select "not applicable"

Even for pure python packages (for example rez), builds can be made reproducible. Generally, non compiled packages will need to be published somewhere and most of the time this will required creating some kind of archive (zip, tar, etc). So packages still have to be build and made reproducible.

Reply:
They can be yes, but this requirement is more focused on distributed compiled binary assets by the project.